### ISSUE TITLE
FIBO's Issue template and labels should be extended and rationalized to represent best practice

### ISSUE THREAD

**Issue by rivettp** [edmcouncil/fibo#837]
To follow various guidelines, make up for loss of fields such as priority which we had in Jira, and allow better alignment with OMG processes.

**Comment by trypuz** [comment]
We should require adding links to the FIBO resources that are discussed in the issue.

**Comment by ElisaKendall** [comment]
I disagree - many of the issues I work on involve 10s or hundreds, in the case of individuals, of resources - this should be a recommendation in cases of small numbers of resources, say, under 10, and not otherwise.

**Comment by ElisaKendall** [comment]
If we are eliminating an ontology, we could include the prefix for that ontology, but should not have to mention all of its resources, for example.

**Comment by rivettp** [comment]
One productivity issue is that the references in the ontology all use prefixes and it becomes very time consuming to replace each by the expanded-out URI. I try to do that for important references but cannot afford the time to do it for everything that might be usefully mentioned in the issue. 

Not to mention that it makes the text of the issue harder to read if it's strewn with URIs which for FIBO tend to be very long: from that point of view the prefixed URIs are more useful.

A minor point, which shouldn't make much difference in practice: the URI will resolve to the published version. Which may differ from, or lag, the latest source in GitHub.

Bottom line, I don't think it should be a rigid rule but a guideline to be applied with judgement.
